positioning-on-background/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,82 @@
+
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
+    <uses-permission android:name="android.permission.CHANGE_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE"/>
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:maxSdkVersion="22" android:name="android.permission.READ_PHONE_STATE"/>
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+
+    <application
+        android:allowBackup="true"
+        android:hardwareAccelerated="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:networkSecurityConfig="@xml/network_security_config"
+        android:theme="@style/AppTheme">
+
+        <activity android:label="@string/app_name" android:name=".BackgroundPositioningActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+
+        <!--Developers should put application credentials here.To obtain them, please register the application
+         at https://developer.here.com/develop/mobile-sdks-->
+        <meta-data
+            android:name="com.here.android.maps.appid"
+            android:value="S4IRz11gyMmsnD0xb1F2" />
+        <meta-data
+            android:name="com.here.android.maps.apptoken"
+            android:value="rICHBw8R1IDarnJ-ebkQ_w" />
+        <meta-data
+            android:name="com.here.android.maps.license.key"
+            android:value="Eips0d95JSoae5bSs+lGzx08KX9FQhpYyExVvKEVtCuIFPQzgGAm3B+cPpIh6Zw4jhe2jHVA5NZ0X88oofyHeWjYlNI8Vw5NcA3kHAQaK/41Brl+frDV2NOtxZOYnOfsFv/pvHeayxWe1iG4iKI3mGacG0dub+8fzf+KduCkpwZH8Zwxii1U87ExrNwZNDuFlDRUsOAzBC3Ah5O+fWSdQWQdwkHQ0h7LIbqW79C9Keu/EkVr1MTaZM+KYcBapg/Kmf3rpab4IWkSh9cR6xdiyhPKp1sRLBmS9SITVegETeQTMSt8HOKopQX6nfH5FjHeH+ITAGt1Txnk+mA2Kri3KiYN6OxMagCh3hxMyyovRN/2CEBeGzG1CF5qkGsrEqhCIg7pprzREgNPTOFR7VG82AOeg1DAs9niUDXRjiOaPp0X1Sf7hqsZ+HTM0sU4nZDgRbH2E4a/3OR+GwxQJq2L+FjYtFouszzQpZkmWtOmcxlC3hr9LRM4eRpYs1spZzoiGjdx5yTRsX2jsKgU3cGDnrayBj1uYZpWaRLkl8xRNhy9DN1q7IEZf7wkh/0rIbjqyb1fcJbWaFmkiinrHqF6fmB8GC2U5d4ccOELtPxp01mBvOCcjCqMz1PENIK7UYFzjXGMZPT93XLv46pmpvVhip8XkBrpP7GbliAyjjacYWQ=" />
```

Comment:
Please remove actual 'values' from commit, set like this https://github.com/heremaps/here-android-sdk-examples/blob/master/advanced-navigation/app/src/main/AndroidManifest.xml#L27-L35

Reply:
Done.